bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueDeviceDiscoveryService.java

```diff
@@ -70,13 +69,13 @@
 public class HueDeviceDiscoveryService extends AbstractDiscoveryService
         implements DiscoveryService, ThingHandlerService {
```

Comment:
Old, but: `AbstractDiscoveryService` implements `DiscoveryService`, so this seems redundant here.

Reply:
Removed.